src/components/PersonLink.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { Person, Sex } from '../types';
+
+type Props = {
+  person: Person;
+  hasMother: Person | null;
+  hasFather: Person | null;
+};
+
+export const PersonLink: React.FC<Props> = ({
+  person,
+  hasMother,
+  hasFather,
+}) => {
+  const {
+    name, slug, sex, born, died, motherName, fatherName,
+  } = person;
```

Comment:
Make wrap everywhere in one code style
```suggestion
  const {
    name,
    slug, 
    sex, 
    born, 
    died, 
    motherName, 
    fatherName,
  } = person;
```